Makefile

```diff
@@ -6,13 +6,11 @@ generate: check-resource
 	go mod tidy; \
 	go run . $(RESOURCE) ;
 
-fmt:
-
 vet:
 	go vet
 
 fmt:
-	go fmt ./...
+	gofmt -s -l -w .
```

Comment:
For context:
`-s` tells `gofmt` to simplify your code
`-l` tells `gofmt` to list the files that changed due to formatting
`-w` tells `gofmt` to write the formatting changes to the source file

Run `gofmt -help` for more info!